docs/user-guide/configuration.md

```
@@ -81,6 +81,40 @@ script: !include scripts.yaml
+picked up automatically, with no operator restart.
+
+!!! warning "The resource is the source of truth"
+    The operator enforces the `http` configuration from the resource. A change
+    made in the Home Assistant UI (Settings → System → Network) to a setting the
+    resource covers is **reverted on the next reconcile** — even though Home
+    Assistant itself points you at that UI. Edit `spec.configuration`, not the
+    Home Assistant UI, for these settings. The operator never confirms a pending
+    change it did not send, so a change you start in the UI is left for you (or
+    Home Assistant's 5-minute auto-revert) to resolve.
+
+    If your `http:` section is an `!include` of a separate file, the operator
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)